src/middleware/sentry.js

```diff
@@ -1,76 +0,0 @@
-/**
- * 3rd party Error Tracking Middleware
- */
-const process = require('node:process');
-const Sentry = require('@sentry/node');
-const {ErrorReporter} = require('../utils/slack.logger');
-const {getImageVersion} = require('../utils/getImageVersion');
-const {getCircularReplacer} = require('../utils/getCircularReplacer');
-
-Sentry.init({dsn: process.env.SENTRY_DSN});
-
-
-process.on('uncaughtException', async (err) => {
-    console.log(
-        JSON.stringify(
-            {
-                method: 'sentryMiddleware.uncaughtException',
-                message: JSON.stringify(err, getCircularReplacer())
-            }
-        )
-    );
```

Comment:
Are we not getting any useful info from these logs?

Reply:
Added back